host/4/bullseye/amd64/dotnet/dotnet8-isolated/dotnet-isolated-composite.template

```diff
@@ -0,0 +1,39 @@
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-preview
```

Comment:
It would be great if you could also tag the specific version of debian that you are using.  Our preferred is Debian 11 Bullseye. But, it doesn't appear that its available for this image.  So explicitly call out that you are using bookworm please

Reply:
Below are the available versions:

- 8.0.0-preview.5-bookworm-slim-amd64, 
- 8.0-preview-bookworm-slim-amd64, 
- 8.0.0-preview.5-bookworm-slim, 
- 8.0-preview-bookworm-slim, 
- 8.0.0-preview.5, 
- 8.0-preview

I wanted to use a version which does not include the preview version (as of today, the latest is preview **5**) ,so that we do not need to update the image again when they release preview 6 (July11). Do you have a recommendation for anything specific?

Reply:
I think that using this would be good.  But please file an issue and link it in a comment near the code.  Then when preview is over we will move to a non-preview build 
8.0-preview-bookworm-slim-amd64

Reply:
Switched to 8.0-preview-bookworm-slim-amd64 and created #927 to remove preview tags.